### DISCUSSION TITLE
Prose wrap preserve doesn´t work

### DISCUSSION THREAD

**Discussion by nixonbohorquez** [prettier/prettier#16440] [Q&A]
I tried in the prettier playground and in my project written with typescript and this still formats my code in a single line, it does not preserve the lines with line breaks, I have my .prettierrc and installed in 3.0.3 like this:
`{
	"arrowParens": "always",
	"bracketSpacing": true,
	"insertPragma": false,
	"printWidth": 120,
	"proseWrap": "preserve",
	"quoteProps": "as-needed",
	"requirePragma": false,
	"semi": true,
	"singleQuote": true,
	"tabWidth": 2,
	"trailingComma": "es5",
	"useTabs": false,
	"endOfLine": "auto"
}`

I would think it is an error since the page does not even format it correctly.

**Comment by cncDAni2** [comment]
Still valid. According to the log, prettier read up the .prettierrc config with the "proseWrap": "preserve", but no matter the value, it always makes my HTML tags into 1 row, if there's a space for that! (I also played with "htmlWhitespaceSensitivity": "ignore"/"strict" - but nothing changes: it always makes my definition back to one line!)
But I think it worked a few month ago btw... (with the same config I have now)
